app.js

```diff
@@ -1,15 +1,14 @@
 const express = require('express');
 const app = express();
 const port = 3000;
-const upload = require('./upload')
-
+const {upload, profile} = require('./upload')
```

Comment:
Use vs code format before commit, I thing you may miss space between `{upload, profile}`